### DISCUSSION TITLE
Semantic search results changed

### DISCUSSION THREAD

**Discussion by fidex111** [milvus-io/milvus#38120] [Q&A and General discussion]
I used to have a good semantic search result when I was using milvus lite. I tried to move to milvus client then the result where not good at all. Now even when I load up my initial script (notebook) that I still have  of my setup of the milvus lite even with same settings/parameter with no modification whatsoever my semantic search result now return the bad search result to be exact same bad answer that the milvus client returns.. what could be the cause . Thanks

**Comment by xiaofan-luan** [comment]
milvus lite is using brute force search(even if you specify ann params), it's mainly built for test. 

So when you turn to milvus, due to the nature of ANN, usually you can only get 90-99% of the recall, some result might be missing.

**Reply by fidex111** [reply]
When I load up the initial milvus lite script in a new notebook and run the semantic search I have done before it now gives me the result of the milvus client.. even when I have released and dropped the collection. What can I do to be getting back the original good search results I was getting in the milvus lite for a while before I decided to implement a milvus client and how can I improve my milvus client results. I have tried to change my index metric type  didn’t really offer much Thanks

**Reply by xiaofan-luan** [reply]
I thinks this is not something milvus lite change. 
More likely you changed your embedding function or preprocessing.
